exercises/practice/armstrong-numbers/armstrong_numbers.zig

```diff
@@ -0,0 +1,3 @@
+pub fn isArmstrongNumber(num: u64) bool {
+    @panic("please implement the isArmstrongNumber function");
+}
```

Comment:
[Non-blocking] What do you think about also setting params to `_` sinks? Zig is a language with opinions so if you don't panic and don't use a variable you'll get errors. I think for the purposes of learning, we might want to add things like:

```
_ = num;
```

It's not strictly necessary for the code to compile, but it might be helpful